### DISCUSSION TITLE
How to properly use desktopCapture

### DISCUSSION THREAD

**Discussion by tonoizer** [electron-vite/electron-vite-vue#285] [Q&A] [answered]
I am quite new to electron and don't know much about it, but wanted to implement a system where you can capture your desktop or any kind of windows and record it. But currently I struggle on how to implement it into this project properly especially with Vue and I could find any resources which show how to do this with Vue.

(Similar to discord when you want to share a screen or application, where you can select in and then record that window/screen)

So, I wondered if anyone could help me out a little bit to get a glimpse on how to implement something like this.

**Comment by caoxiemeihao** [comment, marked as answer]
Maybe this can help you.
https://www.electronjs.org/docs/latest/api/desktop-capturer

**Reply by tonoizer** [reply]
I tried this already and added the snippets into the existing electron folder files. There I wanted to export an array of the screens, but I couldn't get access to them in a Vue file. And in the Vue file I am unable to get the screens.
